.azure-pipelines/build-winrelease.yml

```diff
@@ -26,6 +26,9 @@ steps:
     displayName: Unzipping docs
   - script: |
       @echo on
+      gpg --import create_dmd_release/codesign/dlang.pgp
+      gpg --import create_dmd_release/codesign/nowak.pgp
+      gpg --import create_dmd_release/codesign/wilzbach.pgp
```

Comment:
Uhm you can download them as a keyring: https://dlang.org/gpg_keys.html

That's how the install.sh works as well.

Reply:
Downloading them from external sources seems to somewhat defeat the purpose of verifying the archives. Is it standard to do this?
I tried `--receive-keys` initially but that caused an I/O error on the keyserver.

Reply:
Or do you mean that the keyring file should be added to this repo?

Reply:
Yes, probably a better idea then though as I mentioned earlier I think sha256 hashes might be easier as then everyone with write access can upload files and not only Martin and myself.